pkg/client/cache/fifo.go

```diff
@@ -106,6 +112,10 @@ var (
 	_ = Queue(&FIFO{}) // FIFO is a Queue
 )
 
+func (f *FIFO) Close() {
```

Comment:
I did not consider this fifo to be important. But the implementation here is analogical. I'll clone it.